src/utils.js

```diff
@@ -144,6 +137,11 @@ export function sanityCheck(
     // $FlowFixMe
     err.status = 401;
     return err;
+  } else if (Object.keys(users).length >= maxUsers) {
```

Comment:
Also, I noticed one thing. We have `-1` to disable registration. In that case we have to check that too. 
```
. . .
if(maxUsers !== -1 && Object.keys(users).length >= maxUsers) {
. . .
```

Reply:
@ayusharma I think `Object.keys(users).length >= maxUsers` already handle that. It's a bit sublte but still, returns `409`